addons/web/static/src/js/views/list/list_renderer.js

```diff
@@ -655,6 +653,8 @@ var ListRenderer = BasicRenderer.extend({
                 .append(this._renderGroups(this.state.data))
                 .append(this._renderFooter(true));
         } else {
+            self.hasHandle = self.state.orderedBy.length === 0 ||
+                self.state.orderedBy[0].name === self.handleField;
```

Comment:
i'd do this above, for both grouped and ungrouped cases

Reply:
You're right, it works that way.